### ISSUE TITLE
Memory use on Linux vs Mac

### ISSUE THREAD

**Issue by MaartenSneepKNMI** [jsonpickle/jsonpickle#448]
I have a rather large scikit-learn model, a random forest classifier. The resulting jsonpickle output is about 7.5 GB. This file loads fine on a mac (versions below), but blows up on Linux. Guess where I have to deploy. The script that uses the model loads on the mac into a maximum resident set size of 18.6 GB, while on a linux system the same script requires a rather unwieldy 74 GB (I have access to a machine with 128 GB, but this is not available for deployment). At this moment I'm not even sure if this is an issue in scikit-learn or json pickle. Insights on how to proceed are welcome. Once loaded with sufficient memory the output is identical on both machines. 

scikit-learn version 1.2.2, jsonpickle 3.0.1, python 3.11.3 (on both platforms).

**Comment by Theelx** [comment]
@MaartenSneepKNMI You are correct in that this isn't a jsonpickle issue, however I highly recommend looking into ZRAM on Linux.
